server/src/internalClusterTest/java/org/opensearch/cluster/MinimumClusterManagerNodesIT.java

```diff
@@ -249,6 +249,7 @@ public void testTwoNodesNoClusterManagerBlock() throws Exception {
         }
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/10006")
     public void testThreeNodesNoClusterManagerBlock() throws Exception {
```

Comment:
wrong test?

Reply:
Ugh. Good catch.
Wrong title - correct test. Updated the title